### PR TITLE
Clarifying WiFi connection support

### PR DESCRIPTION
**Pull request by pvernocchi**
Adding a blunt statement that we don't support WiFi connections in MTRs

### CODE REVIEW

File: Teams/rooms/rooms-prep.md [hunk 1]
```diff
@@ -54,7 +54,8 @@ If your network runs through a proxy, you'll need the proxy address or script in
 > Microsoft Teams Rooms does not support proxy authentication as it may interfere with regular operations of the room. Ensure that Microsoft Teams Rooms have been exempted from proxy authentication before going into production.
 
 > [!IMPORTANT]
-> Be sure to use a wired 1 Gbps network connection to assure you will have the needed bandwidth.
+> - Be sure to use a wired 1 Gbps network connection to assure you will have the needed bandwidth.
+> - WiFi connections **are not supported**.
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-prep.md:58]:
```suggestion
> - Wi-Fi connections **are not supported**.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @pvernocchi, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
Wifi statement is not correct. Wifi is supported when it's the only option.

**Comment by pvernocchi** [comment]
Yes, it is now supported. We’ve changed the statement after I submitted the CR.

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request closed without merging**